content/html/concepts/iframes/iframes.md

```diff
@@ -14,13 +14,13 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-IFrame (Inline Frame) is an HTML document embedded inside another HTML document on a website. It provides a smaller version of a browser window that contains various media, such as an advertisement, a YouTube video, etc. Some sites such as Youtube will provide iframe markup for the user to embed instead of making one on their own.
+An **IFrame (Inline Frame)** is an HTML document embedded inside another HTML document on a website. The [`<iframe>`](https://www.codecademy.com/resources/docs/html/tags/iframe) tag provides a smaller version of a browser window that contains various media, such as an advertisement, a YouTube video, etc. Some sites such as Youtube will provide IFrame markup for the user to embed instead of making one on their own.
```

Comment:
```suggestion
An **IFrame (Inline Frame)** is an HTML document embedded inside another HTML document on a website. The [`<iframe>`](https://www.codecademy.com/resources/docs/html/tags/iframe) element provides a smaller version of a browser window that contains various media, such as an advertisement, a YouTube video, etc. Some sites such as Youtube will provide IFrame markup for the user to embed instead of making one on their own.
```